roles/tftp-server/tasks/main.yml

```diff
@@ -1,4 +1,11 @@
 ---
+- name: install a web server to host late_command.sh
+  apt:
+    name: "{{ item }}"
+  with_items:
+  - nginx
+  - python
```

Comment:
heh - that was to install python for ansible when not using late_command.sh. 
which didn't work at all.

added python to preseed.cfg, see below.